### PR TITLE
fix: Update I-D submission instructions. Fix #5264

### LINKED ISSUE #5264: Update submit instructions template
At the very least text like what's at 
https://github.com/ietf-tools/datatracker/blob/5fc93bb8ca243be6aa8c4e848484109e7f3a4d4c/ietf/templates/submit/tool_instructions.html#L142
needs to be changed to match the current reality.
The whole template needs careful review.

### PR DESCRIPTION
**Pull request by russhousley**

### CODE REVIEW

File: ietf/templates/submit/tool_instructions.html [hunk 1]
```diff
@@ -30,15 +30,15 @@ <h2 class="mt-3">I-D submission tool instructions</h2>
     </p>
     <p>
         An API for automated Internet-Draft submission is available as an alternative to this webpage at
-        <a href="{{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.submit.views.api_submit' %}">{{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.submit.views.api_submit' %}</a>.
+        <a href="https://datatracker.ietf.org/api/submission">https://datatracker.ietf.org/api/submission</a>.
```
Comment (rjsparks) [inline ietf/templates/submit/tool_instructions.html:33]: This should use the settings and url reversals. The view is `ietf.submit.views.api_submission`

File: ietf/templates/submit/tool_instructions.html [hunk 2]
```diff
@@ -30,15 +30,15 @@ <h2 class="mt-3">I-D submission tool instructions</h2>
...
     </p>
     <p>
         The specification for this tool can be found in
-        <a href="{% url 'ietf.doc.views_doc.document_html' name='rfc4228' %}">RFC 4228</a>.
+        <a href="{% url 'ietf.doc.views_doc.document_html' name='rfc4228' %}">RFC 4228</a>.  It has been updated over the years.
     </p>
```
Comment (rjsparks) [inline ietf/templates/submit/tool_instructions.html:38]: I think we can just delete this now

File: ietf/templates/submit/tool_instructions.html [hunk 3]
```diff
@@ -60,39 +60,39 @@ <h3 class="mt-4">Upload screen</h3>
...
     <p>
-        After a user uploads the document(s),  the tool will parse the plain-text version, validate the I-D, and display the validation results with option(s) for next steps. The validation includes: checking for all IPR-related notices and I-D boilerplate described in
-        <a href="https://www.ietf.org/ietf/1id-guidelines.html">Guidelines to Authors of Internet-Drafts</a>;
+        After you upload the file(s) and the tool parses the the metadata, the validation results are displayed, and you are offered option(s) for next steps. The validation checks include for IPR-related notices and I-D boilerplate described in
+        <a href="https://www.ietf.org/ietf/1id-guidelines.html">Guidelines to Authors of Internet-Drafts</a>,
         the required sections described in
-        <a href="https://www.ietf.org/ID-Checklist.html">the I-D Check List</a>;
-        the version number; and the creation date.
+        <a href="https://www.ietf.org/ID-Checklist.html">the I-D Check List</a>,
+        the version number, and the creation date.
```
Comment (rjsparks) [inline ietf/templates/submit/tool_instructions.html:89]: I wonder if these should be pointing into the authors wiki instead?

**Review by rjsparks — CHANGES_REQUESTED** [review]

File: ietf/templates/submit/tool_instructions.html (hunk 1 above)
Comment (russhousley) [reply ietf/templates/submit/tool_instructions.html:33]: Sure.  That is fine with me.

File: ietf/templates/submit/tool_instructions.html (hunk 3 above)
Comment (russhousley) [reply ietf/templates/submit/tool_instructions.html:89]: I guess that is a question for Greg Wood.

Comment (rjsparks) [reply ietf/templates/submit/tool_instructions.html:89]: Those URLs all redirect now to https://authors.ietf.org/en/content-guidelines-overview.

File: ietf/templates/submit/tool_instructions.html (hunk 2 above)
Comment (russhousley) [reply ietf/templates/submit/tool_instructions.html]: I removed it.

**Review by rjsparks — APPROVED** [review]

**Pull request merged**